workflow-templates/im-deploy-on-prem-database.yml

```diff
@@ -146,6 +147,8 @@ jobs:
     needs: [set-vars, validate-tag-for-prod-deploys]
     runs-on: [self-hosted, ubuntu-20.04]
     environment: ${{ needs.set-vars.outputs.ENVIRONMENT }}
+    env:
+      VAULT_URL: https://vault.myvault.com:8200
```

Comment:
That's not the actual url for the production vault is it?

related to that, we use the prod vault for storing dev/qa/stage/prod db creds right?